scss/_custom.scss

```diff
@@ -970,10 +970,11 @@ $breadcrumb-padding-y:          .75rem !default;
 $breadcrumb-padding-x:          1rem !default;
 $breadcrumb-item-padding:       .5rem !default;
 
-$breadcrumb-bg:                 $gray-lighter !default;
+$breadcrumb-bg:                 transparent !default;
+$breadcrumb-item-color:         $gray-light !default;
 $breadcrumb-divider-color:      $gray-light !default;
 $breadcrumb-active-color:       $gray-light !default;
-$breadcrumb-divider:            "/" !default;
+$breadcrumb-divider:            " " !default;
```

Comment:
"_blank" in the ticket description was a mark to ask @maxstrebel for the character he'd like to use as a breadcrumb separator